.github/workflows/publish.yml

```diff
@@ -43,3 +44,174 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+  build_binaries:
+    strategy:
+      matrix:
+        #unless we want windows binaries as well, keep the matrix as is
+        os: [ubuntu-latest, macos-latest]
+        #os: [ubuntu-latest, windows-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      #for windows only, use correct newlines
+      - name: Set git to use LF
+        if: matrix.os == 'windows-latest'
+        run: |
+          git config --global core.autocrlf false
+          git config --global core.eol lf
+
+      - uses: actions/checkout@v3
+
+      # Check Cargo.toml in sqld sbdir has the correct tag
+      - name: Verify sqld/Cargo.toml version is same as git tag
+        if: ${{ startsWith(github.ref, 'refs/tags/') }} && ${{ matrix.os }} == ubuntu-latest
+        id: check-tag
+        run: |
+          VER=$(echo ${{ github.ref_name }}|tr -d "v")
+          pushd sqld
+          CARGOVER=$(cargo metadata --no-deps | jq '.packages | .[] | select(.name == "sqld") | .version'|tr -d '"')
```

Comment:
will this work before you install a toolchain?

Reply:
also, is jq already installed?

Reply:
jq is there (see https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md) but the order needs to be changed, I'll fix it